### PR TITLE
637 Updated converter product detail page

### LINKED ISSUE #637: [Product] Converter update product detail page

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #637 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://637-converter-update-product-page--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
Looks good to me, but @mhaack  should probably review.

File: tools/actions/convert/test/fixtures/product2-converted.html [hunk 1]
```diff
@@ -14,27 +14,32 @@
         </div>
       </div>
       <hr>
-      <div class="product-details">
+      <div class="product-hero">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/product2-converted.html:17]:
```suggestion
      <div class="Product Hero">
```

File: tools/actions/convert/test/fixtures/product2-converted.html [hunk 2]
```diff
@@ -14,27 +14,32 @@
...
         <div>
           <div>Request for Quote</div>
         </div>
       </div>
       <hr>
+      <div class="feature">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/product2-converted.html:23]: What is this for?

File: tools/actions/convert/test/fixtures/product2-converted.html [hunk 3]
```diff
@@ -177,7 +182,12 @@ <h2>Enhanced for stem cell applications*</h2>
         </div>
       </div>
       <hr>
-      <div class="product-attribute-table">
+      <div class="specification">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/product2-converted.html:185]:
```suggestion
      <div class="Product Specification">
```

File: tools/actions/convert/test/fixtures/product2-converted.html [hunk 4]
```diff
@@ -198,6 +208,11 @@ <h2>Enhanced for stem cell applications*</h2>
         </div>
       </div>
       <hr>
+      <div class="products">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/product2-converted.html:211]:
```suggestion
      <div class="Related Products">
```

File: tools/actions/convert/test/fixtures/product2-converted.html [hunk 5]
```diff
@@ -209,6 +224,11 @@ <h2>Enhanced for stem cell applications*</h2>
         </div>
       </div>
       <hr>
+      <div class="resources">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/product2-converted.html:227]:
```suggestion
      <div class="Product Resources">
```

File: tools/actions/convert/test/fixtures/product2-converted.html [hunk 6]
```diff
@@ -220,6 +240,11 @@ <h2>Enhanced for stem cell applications*</h2>
         </div>
       </div>
       <hr>
+      <div class="citations">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/product2-converted.html:243]:
```suggestion
      <div class="Product Citations">
```

File: tools/actions/convert/test/fixtures/product2-converted.html (hunk 3 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/product2-converted.html]: Done

File: tools/actions/convert/test/fixtures/product2-converted.html (hunk 5 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/product2-converted.html]: Done

File: tools/actions/convert/test/fixtures/product2-converted.html (hunk 4 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/product2-converted.html]: Done

File: tools/actions/convert/test/fixtures/product2-converted.html (hunk 6 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/product2-converted.html]: Done

File: tools/actions/convert/test/fixtures/product2-converted.html (hunk 2 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/product2-converted.html]: Updated as discussed

**Review by duynguyen — APPROVED** [review]

**Pull request merged**